examples/sample-pub/sample-pub-app.js

```diff
@@ -334,6 +334,29 @@ app.get('/redirect-to/swg-basic.js', (req, res) => {
   res.redirect(SWG_BASIC_JS_URLS[setup.script]);
 });
 
+/**
+ * EA - Simplified API
+ * TODO: make it a flow of "An Article"
+ */
+ app.get('/ea-simplified-api', (req, res) => {
+  const id = 1;
+  const article = ARTICLES[id - 1];
+  const prevId = id - 1 >= 0 ? String(id - 1) : false;
+  const nextId = id + 1 < ARTICLES.length ? String(id + 1) : false;
```

Comment:
Explain how this differs from production usage

Reply:
Redundant code, removed.